### PR TITLE
Clarify discussion regarding solar position being calculated for the pixel center

### LINKED ISSUE #3: Solar position at center of pixel
`pages/solar-position` includes the claim:

> 4. The solar zenith angle returned from the API is calculated for the center of the PSM3 pixel.

This is not necessarily supported by the notebook's current content.  Need to verify the claim and adjust the page accordingly.

### PR DESCRIPTION
**Pull request by kanderso-nrel**
Closes #3.

### CODE REVIEW

File: source/pages/solar-position.ipynb [hunk 1]
```diff
@@ -12,12 +12,12 @@
     "1) The solar position algorithm used by the API is equivalent to pvlib's implementation of the SPA to within the precision of the values returned from the API (2 digits after the decimal).\n",
```
Comment (AdamRJensen) [inline source/pages/solar-position.ipynb:12]: <p>Is there a space missing between "the" and "true"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (kanderso-nrel) [reply source/pages/solar-position.ipynb:12]: I'm not following.  Maybe I'm missing something, but I don't see anything matching this description in this notebook.  (I'm looking at this on GitHub, not on ReviewNB)

Comment (AdamRJensen) [reply source/pages/solar-position.ipynb:12]: Could just be me incorrectly viewing the notebook, but I see this using ReviewNB:
image

It seems just to be an artifact, as it's not there when I look at the notebook standalone.

**Review by AdamRJensen — APPROVED** [review]
Nice clarifying addition! Great job @kanderso-nrel 

Feel free to merge.

**Pull request merged**